src/components/pages/Footer.astro

```diff
@@ -148,8 +157,15 @@ import WmNamedLogo from '@assets/wm-named-logo.svg'
         <WmNamedLogo />
       </a>
     </div>
-    <small>
-      Copyright © {new Date().getFullYear()} Interledger Foundation.
+    <small class='copyright'>
+      <a href='https://www.w3.org/policies/#copyright'>Copyright</a> © {
+        new Date().getFullYear()
+      } the Contributors to the Web Monetization Specification, published by the
+      <a href='https://www.w3.org/community/wicg/'
+        >Web Platform Incubator Community Group</a
+      > under the <a href='https://www.w3.org/community/about/process/cla/'
+        >W3C Community Contributor License Agreement (CLA)</a
+      >. A human-readable summary is available.
```

Comment:
`summary` needs to be linked to http://www.w3.org/community/about/agreements/cla-deed/.